Source/WebInspectorUI/UserInterface/Views/TreeOutline.js

```diff
@@ -316,13 +316,19 @@ WI.TreeOutline = class TreeOutline extends WI.Object
         let child = this.children[childIndex];
         let parent = child.parent;
 
-        if (child.deselect(suppressOnDeselect) && !suppressSelectSibling) {
-            if (child.previousSibling)
-                child.previousSibling.select(true, false);
-            else if (child.nextSibling)
-                child.nextSibling.select(true, false);
+        let currentSelectedElement = this.treeOutline.selectedTreeElement;
```

Comment:
How does this work when multiple things are selected?  Maybe we should just check `descendant.selected` instead?